iTechArt.Repositories/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace iTechArt.Repositories.Interfaces
+{
+    public interface IRepository<T> where T : class
+    {
+        IEnumerable<T> GetAll();
```

Comment:
1. Dont ever return enumerables. Replace with `IReadOnlyCollection<T>`.
2. Convert method to async. Add *Async suffix.
3. Add Where method for filtering.